### PR TITLE
add: home page

### PR DESCRIPTION
**Pull request by menuason**
Screenshot 2024-02-23 at 2 28 19 PM
Screenshot 2024-02-23 at 2 28 28 PM
Screenshot 2024-02-23 at 2 28 07 PM

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -1,36 +1,37 @@
-import logo from '../assets/logo/logo-dark.svg';
-import { useTranslation } from 'react-i18next';
-import MultiLanguageSelect from './MultilanguageSelect';
-import HamburgerSelect from './HamburgerSelect';
+import logo from "../assets/logo/logo-dark.svg";
+import { useTranslation } from "react-i18next";
+import MultiLanguageSelect from "./MultilanguageSelect";
+import HamburgerSelect from "./HamburgerSelect";
```
Comment (evosliza) [inline src/components/Header.tsx]: please make sure you have the same settings, so we don't have these changes in every pr

File: src/pages/HomePage/AboutBloomHouse.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import arrowRight from "../../assets/icons/arrowRight/arrowRight.svg";
+
+export const AboutBloomHouse = () => {
+  return (
+    <div className="h-[911px] flex flex-row gap-16 px-20">
```
Comment (evosliza) [inline src/pages/HomePage/AboutBloomHouse.tsx]: are we sure `h-[911px]` needs to be this way?

File: src/pages/HomePage/Assortment.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+const flowersArr = [
+  {
+    key: 1,
```
Comment (evosliza) [inline src/pages/HomePage/Assortment.tsx]: let's do not have `key` prop and use array index instead

File: src/pages/HomePage/WhyCoosUs.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import temperature from "../../assets/icons/iconsForChoosUs/Temperature.svg";
+import timeCosuming from "../../assets/icons/iconsForChoosUs/timeCosuming.svg";
+
+const iconsForChoosUs = [
+  { icon: timeCosuming, text: "Best Seedlings" },
+  { icon: temperature, text: "Precision Cultivation" },
+  { icon: precisionCultivation, text: "Temperature Control" },
+];
+
+const infoForChoosUs = [
```
Comment (evosliza) [inline src/pages/HomePage/WhyCoosUs.tsx]: you have a typo here

File: src/pages/HomePage/WhyCoosUs.tsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  },
+  {
+    number: 3,
+    header: "Temperature Control",
+    description:
+      "The perfect temperature for gerberas is ensured through our advanced control systems, ensuring they thrive in a comfortable and conducive environment.",
+  },
+];
+
+export const WhyChoosUs = () => {
```
Comment (evosliza) [inline src/pages/HomePage/WhyCoosUs.tsx]: you have a typo here

File: src/pages/HomePage/WhyCoosUs.tsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        bloom of vibrant colors and exquisite blossoms.
+      </h6>
+      <div className="flex mt-9 ">
+        <div className="flex flex-col m-0">
+          <img src="src/assets/home-page-photos/5P4A3279.png" />
+          <div className="h-40 w-640 flex items-center justify-between">
+            {iconsForChoosUs.map((item) => {
+              return (
+                <div
+                  key={item.icon}
```
Comment (evosliza) [inline src/pages/HomePage/WhyCoosUs.tsx]: use index as `key`

**Review by elenamkoian — APPROVED** [review]

**Review by elenamkoian — APPROVED** [review]

File: src/App.tsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import Header from "./components/Header";
 import Footer from "./components/Footer";
+import HomePage from "./pages/HomePage/HomePage";
```
Comment (evosliza) [inline src/App.tsx:4]: you have index.ts, why do you import from here?

**Pull request merged**